app/Models/User.php

```diff
@@ -65,7 +65,15 @@ public function associacoesPresididas()
     {
         return $this->belongsToMany(Associacao::class, 'associacao_presidente', 'presidente_id', 'associacao_id')->withTimestamps();
     }
+    public function associacoesSecretario()
+    {
+        return $this->belongsToMany( Associacao::class, 'associacao_secretario', 'secretario_id', 'associacao_id')->withTimestamps();
+    }
 
+    public function oscAgricultores()
```

Comment:
Novamente a relação many to many.
Vamos lá:

1. Acredito que era para ser ocsAgricultores.
2. Se a relação é entre usuário e ocs, no modelo do usuário coloca-se o nome do método como apenas ocs.
3. Essa relação já existe, é a "organizacao" mais em baixo.
4. A método indica uma relação com o ocs mas a classe que está sendo usada é do model de Associacao.
5. Ao criar uma tabela intermediária, por convenção colocamos os nomes das classes em ordem alfabetica e no singular. Ficaria agricultor_ocs, ou agricultor_organizacao.